src/plugins/LADTable/components/LADRow.vue

```diff
@@ -58,10 +56,16 @@ export default {
             currentObjectPath
         }
     },
+    computed: {
+        formattedTimestamp() {
+            return this.timestamp !== '---' ? this.formats[this.timestampKey].format(this.timestamp) : this.timestamp;
```

Comment:
Rather than having a variable that is either a number or a string (which I found a little confusing), it might be clearer if this.timestamp was undefined until it was set. `formattedTimestamp()` could then return `---`.

Reply:
Makes sense. I've updated timestamp to be initially set to undefined, then we figure out what to return in formattedTimestamp.